pysrc/bytewax/bigquery/bigquery_output.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from google.cloud import bigquery
+
+from bytewax.outputs import ManualOutputConfig
+
+
+def output_builder(table_ref):
+    """Constructs an output function to write to BigQuery"""
+    client = bigquery.Client()
+    table = client.get_table(table_ref)
+
+    def write_rows(rows):
+        errors = client.insert_rows_json(table, rows)  # Make an API request.
```

Comment:
What do you think about repeating the concept of https://github.com/bytewax/bytewax/pull/152#discussion_r993736853 ?

I think it's a cool trick to tell people to format their items as the kwargs to be passed to the library's writing function. Then they can use the fancier features of the library without us having to manually thread them through. It's a little bit of a balance in that we still don't support any arbitrary use of the library, like an output process that might involve other functions on `client`, but it does allow a lot more flexibility than only allowing the most vanilla of writes.

So something like `client.insert_rows_json(table=table, **kwargs)`.